### PR TITLE
Revert "Make WMCO azure tests optional"

### PR DESCRIPTION
**Pull request by sebsoto**
This reverts commit 30e561a1021255bc9b4716336b19fe3e2d5a73cc.
This is being done as we have now made it so that azure e2e tests are
working properly.
https://github.com/openshift/windows-machine-config-operator/pull/106

### CODE REVIEW

**Comment by sebsoto** [comment]
/hold
hold should be removed once the mentioned PR merges

**Review by aravindhp — APPROVED** [review]
/lgtm

**Pull request merged**
